packages/cli/src/runAndroid/adb.js

```diff
@@ -34,16 +34,43 @@ function parseDevicesResult(result: string): Array<string> {
 /**
  * Executes the commands needed to get a list of devices from ADB
  */
-function getDevices(): Array<string> {
+function getDevices(adbPath: string): Array<string> {
   try {
-    const devicesResult = execSync('adb devices');
+    const devicesResult = execSync(`${adbPath} devices`);
     return parseDevicesResult(devicesResult.toString());
   } catch (e) {
     return [];
   }
 }
 
+/**
+ * Gets available CPUs of devices from ADB
+ */
+function getAvailableCPUs(adbPath: string, device: string): Array<string> {
+  try {
+    const baseArgs = ['-s', device, 'shell', 'getprop'];
+
+    let cpus = execFileSync(
+      adbPath,
+      baseArgs.concat(['ro.product.cpu.abilist'])
+    ).toString();
+
+    // pre-Lollipop
+    if (!cpus || cpus.trim().length === 0) {
+      cpus = execFileSync(
+        adbPath,
+        baseArgs.concat(['ro.product.cpu.abi'])
+      ).toString();
+    }
+
+    return (cpus || '').trim().split(',');
+  } catch (e) {
```

Comment:
What about logging what happened?

Reply:
I think it won't hurt? Feel free to use `logger.error()` instead of `console.log`, with a message that there was an error but we continue with some defaults.

Reply:
@thymikee `logger.error` was added. Thank you for your review.